data-prepper-plugins/service-map-stateful/src/test/java/com/amazon/dataprepper/plugins/prepper/ServiceMapTestUtils.java

```diff
@@ -101,4 +104,43 @@ public static ExportTraceServiceRequest getExportTraceServiceRequest(ResourceSpa
                 .build();
     }
 
+    /**
+     * Creates a {@link com.amazon.dataprepper.model.trace.Span} object with the given parameters
+     * @param serviceName Resource name for the ResourceSpans object
+     * @param spanName Span name for the single span in the ResourceSpans object
+     * @param spanId Span id for the single span in the ResourceSpans object
+     * @param parentId Parent id for the single span in the ResourceSpans object
+     * @param spanKind Span kind for the single span in the ResourceSpans object
+     * @return {@link com.amazon.dataprepper.model.trace.Span} object with a single span constructed according to the parameters
+     */
+    public static Span getSpan(final String serviceName, final String spanName, final String
+            spanId, final String parentId, final String traceId, final io.opentelemetry.proto.trace.v1.Span.SpanKind spanKind) {
+        final String endTime = UUID.randomUUID().toString();
+        final JacksonSpan.Builder builder = JacksonSpan.builder()
+                .withSpanId(spanId)
+                .withTraceId(traceId)
+                .withTraceState("")
+                .withParentSpanId(parentId)
+                .withName(spanName)
+                .withServiceName(serviceName)
+                .withKind(spanKind.name())
+                .withStartTime(UUID.randomUUID().toString())
```

Comment:
I guess our tests don't really validate time. But, it would probably be better to create a time.

Something like this can work:

```
Instant.now().minusMinutes(10).toString();
```

You could make the `10` somewhat random, but that probably doesn't matter much.